### PR TITLE
Feat: Added Button Component 

### LINKED ISSUE #9: Button Components
# Button Components
- Must have feature (P0)

## Checklist 
- [x] Primary, Info, Success, Warning, Danger
- [x] Button Variants
- [x] Link
- [x] Icon Button
- [x] Floating Action Button

### PR DESCRIPTION
**Pull request by pnchinmay**
#### PULL REQUEST TYPE

- Improvement/feature implementation

#### CHECKLIST

- [x] Added button components
- [x] Added button styling classes

#### LINKED ISSUES

resolves #9

### CODE REVIEW

File: pages/button.html [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                    <a href=""></a>
+                    <a href="/pages/introduction.html">
+                        <i class="fas fa-angle-left"></i> Installation</a>
+                    </div>
+                    <div>
+                    <a href=""></a>
+                    <a href="/pages/avatar.html">
+                        Avatar <i class="fas fa-angle-right"></i></a>
+                    </div>
+                </footer> -->
```
Comment (sharmayatendra) [inline pages/button.html]: remove commented code

**Review by sharmayatendra — APPROVED** [review]
good to go

File: pages/button.html (hunk 1 above)
Comment (pnchinmay) [reply pages/button.html]: Done

File: pages/button.html [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+                    <div>
+                    <a href=""></a>
+                    <a href="/pages/introduction.html">
+                        <i class="fas fa-angle-left"></i> Installation</a>
+                    </div>
+                    <div>
+                    <a href=""></a>
+                    <a href="/pages/avatar.html">
+                        Avatar <i class="fas fa-angle-right"></i></a>
+                    </div>
```
Comment (vikrantkode) [inline pages/button.html]: try to remove commented code while raising PR

**Review by vikrantkode — COMMENTED** [review]
Minor change suggested. Otherwise good to go
- UI is clean

**Pull request merged**